ompi/mca/pml/ucx/pml_ucx.h

```diff
@@ -59,6 +59,7 @@ struct mca_pml_ucx_module {
     int                       priority;
     bool                      cuda_initialized;
     bool                      request_leak_check;
+    uint32_t                  op_attr_nonblocking;
```

Comment:
Why is it `uint32_t` and not `bool`? The MCA parameter used to set it is bool (https://github.com/open-mpi/ompi/pull/9610/files#diff-6a2952e2730c99111221330017fc798631ad6603c29a257ce3bb114e588f5984R90).

Reply:
In future it can hold another flags for non-blocking operations.